src/newWizard/Wizard.stories.tsx

```diff
@@ -104,6 +105,26 @@ const store = new Store({
         store.set({
             open: false,
         }),
+    handleSyncComplete: (): void => {
+        action("complete with loading spinner") &&
+            store.set({
+                isApplying: true,
```

Comment:
Can we keep it showSpinner instead.  The usecase of disabling leftNav and buttons is not only limited to apply button.  We can use it for 
1) Sizer calls
2) Template calls
3) Verify BucketName, Object Store names (usercases coming in future)

So lets keep it generic.

Reply:
I will raise different PR for this